ocs_ci/ocs/ui/base_ui.py

```diff
@@ -1116,12 +1120,16 @@ def navigate_to_all_clusters(**kwargs):
         timeout = 30
 
     local_clusters_dropdown = acm_page["local-cluster_dropdown"]
+    find_element = wait_for_element_to_be_visible(acm_page["click-local-cluster"], 60)
     try:
-        acm_dropdown = wait_for_element_to_be_visible(local_clusters_dropdown, timeout)
-        acm_dropdown.click()
-        all_clusters_item = wait_for_element_to_be_visible(
-            acm_page["all-clusters_dropdown_item"]
-        )
-        all_clusters_item.click()
+        if not not find_element:
```

Comment:
due to implementation of wait_for_element_to_be_visible, it will fail if element not found.
The if not not find_element will not work.

Also "not" has been used 2 times
```
def wait_for_element_to_be_visible(locator, timeout=30):
    """
    Wait for element to be visible. Use when Web element is not have to be clickable (icons, disabled btns, etc.)
    Method does not fail when Web element not found

    Args:
         locator (tuple): (GUI element needs to operate on (str), type (By)).
         timeout (int): Looks for a web element until timeout (sec) occurs

    Returns:
        selenium.webdriver.remote.webelement.WebElement: Visible web element.
    """
    wait = WebDriverWait(SeleniumDriver(), timeout)
    try:
        web_element = wait.until(
            ec.visibility_of_element_located((locator[1], locator[0]))
        )
    except TimeoutException:
        take_screenshot()
        copy_dom()
        raise
    return web_element
```